stats_test.go

```diff
@@ -103,3 +104,78 @@ func TestNilAggregation(t *testing.T) {
 		t.Errorf("Expected: %v, Got: %v", fmt.Errorf("aggregation *view.Aggregation is not supported"), actual)
 	}
 }
+
+func Test_calculatePercentile(t *testing.T) {
+	var buckets []float64
+	for i := float64(-100); i < 100; i += 0.1 {
+		buckets = append(buckets, i)
+	}
+
+	normalDistribution := calculateNormalDistribution(buckets, 0, 100)
```

Comment:
I would suggest using a fixture data instead of generated test data.
That makes the tests easier to understand and to verify the correctness of the test.

For example, in this test, it could be challenging to check if the expected value is indeed what we expect.

Some interesting examples here: https://github.com/montanaflynn/stats/blob/master/percentile_test.go

Reply:
I don't necessarily agree with this one, although it's generating data, it's a pretty standard data set that is guaranteed to be the same each run because we are using the same random seed. In addition the percentiles that are calculated, can be verified with a standard normal table such as the table on this wiki article https://en.wikipedia.org/wiki/Standard_normal_table

Reply:
I think this is fine, but it would be good (and essential, IMO) to also have a test for `calculateNormalDistribution` in this case. Some extra documentation by means of comments would also be nice.

Reply:
First, thank you a lot for the updates you made.

Got a question about `Test_calculatePercentile`, how the `expected` numbers `0, 44, 86, 97, 99` have been calculated/generated?

Reply:
Hi folks, just follow up this this PR.

One alternative would be using an external library to compute the Percentile so we don't have to implement/maintain/test our own version of it ? This seems a good candidate https://github.com/montanaflynn/stats

Reply:
If the current algorithm works, I don't see why we'd want to pull in any dependencies to this project which currently has none 👍(other than `tinylib/msgp`)

Reply:
Yes, that's a good points, avoiding dependencies is a 👍.

About the currently algorithm, my only concern is about having tests that are easy to understand and to maintain (the initial feedback of this thread + [this question](https://github.com/DataDog/opencensus-go-exporter-datadog/pull/41#discussion_r285681575)).

Reply:
Agreed. Tests need to be made crystal clear.

Reply:
I've changed the tests to use a standard deviation of 1, which results in smaller percentiles and makes a bit harder to compare the actuals vs the gots. But it results in being able to use a standard normal table to validate the resulting percentiles are correct.